Lesson23/Mesut/filter.js

```diff
@@ -353,8 +376,13 @@ const studentsExercise9 = [
 	},
 ];
 
+const studentsExercise9Result=studentsExercise9.filter(student=>student.grade=="A"||student.grade=="B"||student.grade=="C")
```

Comment:
```suggestion
const studentsExercise9Result=studentsExercise9.filter(student=>student.grade=="A"||student.grade=="B"||student.grade=="C");
```

Good but it would be even better to cover grades like (A-, B+ etc.)